readthedocs.yml

```diff
@@ -1,5 +1,4 @@
 conda:
-    file: docs/environment.yml
+    file: docs/conda_env.yml
```

Comment:
No problem. We've used both conda_env.yml in most of the other repos but I think going with the standard environment.yml makes more sense.